subject_files_status_for_dpdash.py

```diff
@@ -94,13 +110,43 @@ def phoenix_files_status(phoenix_dir, out_dir):
             'weekday': ''
         })
         
-        subject_df_tmp = pd.concat([subject_df_tmp, pd.DataFrame(subject_series_tmp).T], axis=1)
-        
+        subject_df_tmp = pd.concat(
+                [subject_df_tmp, pd.DataFrame(subject_series_tmp).T],
+                axis=1)
         out_file = f"{site[-2:]}-{subject}-flowcheck-day1to1.csv"
-
         subject_df_tmp.to_csv(out_dir/out_file, index=False)
 
 
+def _get_count(root: Path, subdir: str, pattern: str) -> int:
+    '''get number of files and directories that mattch glob pattern'''
+    return len(list((root / subdir).glob(str)))
+
+
+def _get_dir_count(root: Path, subdir: str, pattern: str) -> int:
+    '''get number of directories that mattch glob pattern'''
+    return len([x for x in list((root / subdir).glob(str)) if x.is_dir()])
+
+
+def _get_file_count(root: Path, subdir: str, pattern: str) -> int:
+    '''get number of files that mattch glob pattern'''
+    return len([x for x in list((root / subdir).glob(str)) if x.is_file()])
+
+
+def _is_file(root: Path, subdir: str, pattern: str) -> int:
```

Comment:
`is_file`, `is_dir`, and `is_scansheet` return types should be `bool`

Reply:
[fd8efc7](https://github.com/AMP-SCZ/utility/pull/7/commits/fd8efc79b7410ce999f931a5c43b4a3accabc9e6)